packages/core/src/integration.ts

```diff
@@ -10,46 +16,64 @@ export type IntegrationIndex = {
 };
 
 /**
+ * Remove duplicates from the given array, preferring the last instance of any duplicate. Not guaranteed to
+ * preseve the order of integrations in the array.
```

Comment:
```suggestion
 * Remove duplicates from the given array, preferring the last instance of duplicate instances, and user instances over
 * default instances. Not guaranteed to preseve the order of integrations in the array.
```

Reply:
Technically? Yes. But I'd argue that 

a) it falls under the heading of ["fixing bugs is always technically breaking"](https://xkcd.com/1172/), and 

b) coming back to the examples from the description, we have
- `[UserProvidedCopyofIntegration, DefaultCopyOfSameIntegration]` - the user's copy should win, and already does (no behavior change here)
- `[DefaultCopyofIntegration, UserProvidedCopyOfSameIntegration]` - the user's copy should win, but currently doesn't (this seems wrong on its face and also, why would anyone rely on us ignoring their copy?)
- `[DefaultCopyAofIntegration, DefaultCopyBofIntegration]` - copy B should win, but currently doesn't (we have total control here, though I don't think it actually comes up anywhere)
- `[UserCopyAofIntegration, UserCopyBofIntegration]` - copy B should win, but currently doesn't (the one breaking, not-currently-quite-as-obviously-wrong change, but I'm wagering that the number of people actively providing two copies of the same integration is vanishingly small)

I honestly have no idea why we were preferring first to begin with (and [the PR](https://github.com/getsentry/sentry-javascript/pull/1646) which introduced both the logic and the test unfortunately says absolutely nothing about it), so I feel okay changing it.

WDYT?

Reply:
lol at the xkcd

I am totally fine with the changes. If people report in that they see problems, we can tell them to change the order to match the fixed behavior!

Reply:
Whoops - completely forgot to include your good suggestion above before I merged. I'll throw it into the pile for the next comments-fix PR.